main.tf

```diff
@@ -4,6 +4,7 @@ data "ssh_tunnel" "consul" {
   user            = "stefan"
   host            = "bastion.example.com"
   private_key     = "${file(pathexpand("~/.ssh/id_rsa"))}"
+//  ssh_agent = true  // use ssh-agent socket
```

Comment:
The `//` should be after two spaces. And I feel like the name of the attribute is descriptive enough so that the comment at the end is unnecessary.

Reply:
Rewritten the TF example to show the private and ssh-agent key usage